.github/workflows/release.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Build project
         run: |
           mkdir out
-          CGO_ENABLED=0 go build -a -installsuffix cgo -o ./out/terratag/terratag
+          CGO_ENABLED=0 go build ./cmd/terratag -a -installsuffix cgo -o ./out/terratag/terratag
```

Comment:
This change I cannot test.
When releasing a new terratag version need to make sure I didn't break anything in the release process.

Reply:
did you run this command localy ?

Reply:
Good point!
I actually found a problem see:
d48e230ba7262eff734776917c4adc383a0e3b69